usr/src/uts/common/os/log_sysevent.c

```diff
@@ -641,6 +642,10 @@ sysevent_add_attr(sysevent_attr_list_t **ev_attr_list, char *name,
 		error = nvlist_add_hrtime(*ev_attr_list, name,
 		    se_value->value.sv_time);
 		break;
+	case SE_DATA_TYPE_NVLIST:
```

Comment:
In illumos/Solaris, some sysevent consumers only look at a single-level nvlist. In practice, adding multiple-level nvlists is of questionable value due to the need to remain compatible with older consumers. See syseventadm(1m) for an example of such a limited consumer. Since syseventadm is used by third parties to implement other consumers, in practice it is simpler to keep a single-level nvlist.